src/app/modules/lesson/lesson.component.ts

```diff
@@ -18,6 +18,7 @@ export const GET_LESSON = gql`
         lessonTitle
       }
       references
+      instructor
     }
     words(
       where: { wordId_in: ["nextLesson", "continue", "greatJobLesson", "back"] }
```

Comment:
You can create a new entry in GraphCMS, under Content => Word. You could call it `writtenBy`, add it both in Swedish  and English, and add it to this array. 
You can then use it in `lesson.component.html` instead of hardcoding it in Swedish :)